data-processing/common/commands/fetch_arxiv_sources.py

```diff
@@ -47,14 +53,34 @@ def load(self) -> Iterator[ArxivId]:
         for arxiv_id in self.arxiv_ids:
             yield arxiv_id
 
-    def process(self, item: ArxivId) -> Iterator[None]:
+    def process(
+        self,
+        item: ArxivId
+    ) -> Iterator[None]:
         if self.args.source == "arxiv":
-            fetch_from_arxiv(item)
+            attempt = 0
```

Comment:
do we not have a little python util for wrapping things in this kinda retry logic? At BV for instancewe had a python function that took:
- function to call
- optional[exceptions to retry on[]]
- backoff delay

Which made it really easy to throw into places

Reply:
We do not have such a utility. It would be good to put in `scholar-data-util`, but that's a private repo we can't depend on here (even though we could copypasta the code).